redaxo/src/addons/media_manager/lib/effects/effect_image_format.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+/**
+ * @package redaxo\media-manager
+ */
+
+class rex_effect_image_format extends rex_effect_abstract
+{
+    private static $convert_types = [
+        'jpg',
+        'png',
+        'gif',
+        'webp',
+    ];
+
+    private static $convert_to = [
+        'jpg' => [
+            'ext' => 'jpg',
+            'content-type' => 'image/jpeg',
+        ],
+        'png' => [
+            'ext' => 'png',
+            'content-type' => 'image/png',
+        ],
+        'gif' => [
+            'ext' => 'gif',
+            'content-type' => 'image/gif',
+        ],
+        'webp' => [
+            'ext' => 'webp',
+            'content-type' => 'image/webp',
+        ],
+    ];
+
+    private static $convert_tos = ['jpg', 'png', 'gif', 'webp'];
+    private static $convert_to_default = 'webp';
+
+    public function execute()
+    {
+        $media = $this->media;
+
+        $ext = $media->getFormat();
+        $ext = strtolower('jpeg' === $ext ? 'jpg' : $ext);
```

Comment:
```suggestion
        $ext = strtolower($media->getFormat());
        $ext = 'jpeg' === $ext ? 'jpg' : $ext;
```

Mir ging es darum, wenn die Extension theoretisch `JPEG` großgeschrieben wäre, dass es dann auch funktionieren soll.
Deswegen das strtolower noch vor dem jpeg-check.